teachopencadd/talktorials/T018_automated_cadd_pipeline/talktorial.ipynb

```diff
@@ -1956,7 +1956,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<a id='Binding-site-detection'></a>\n",
+    "<a id='Binding-site-detection_'></a>\n",
```

Comment:
In the latest version of the talktorial I pushed, I've changed the name of two sections (in Theory: "Binding site detection" -> "Protein binding site", in Practical: "Molecular docking" -> "Docking calculations") so that sections in Theory and Practical don't have the same name (so that we can use cell-IDs matching the title of the cell in order to avoid problems in the HTML rendering, because I remember you said they should have matching names). Nevertheless we should also check all cross-references after rendering to HTML. As I said, unfortunately I don't know how we do that.